package.json

```diff
@@ -53,5 +56,17 @@
     "example": "opn http://localhost:3000/example/ && serve -p 3000 .",
     "prepublish": "npm run build",
     "release": "npmpub"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm test && lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.js": [
+      "eslint --fix **/*.js",
```

Comment:
File will be passed as argument, so you don't need to send `**/*.js`. Just `"eslint --fix",